docs/Proxies.md

```diff
@@ -77,7 +77,7 @@ In the case of contract upgrades, it is achieved by simply changing the proxy's
 
 ### Proxy contract
 
-The [Proxy contract](../src/openzeppelin/upgrades/Proxy.cairo) includes two core methods:
+The [Proxy contract](../src/openzeppelin/upgrades/proxy/presets/Proxy.cairo) includes two core methods:
```

Comment:
mhh unsure about proxy living under upgrades. I know it did before, but it now carries different semantics since "proxy" in the path now looks like a library/group rather than a mere preset serving a role in the context of upgrades.

Maybe switching the order and placing upgrades as a special case for proxies? not sure

Reply:
Along those same lines, what if we remove "upgrades" altogether?

`openzeppelin/proxy/library.cairo`
`openzeppelin/proxy/presets/Proxy.cairo`

Reply:
what about removing proxy? this is about upgrades (as the events and library indicate)
```
openzeppelin/upgrades/library.cairo
openzeppelin/upgrades/presets/Proxy.cairo
```